src/guides/getting-started.md

```diff
@@ -573,6 +573,59 @@ That's it! Let's try it out with `cargo run --bin publish_post 1`.
 Published post Diesel demo
 ```
 
+Additionally let's implement a possibility of fetching a single post. We will display the post id with its title.
+Notice [`.optional()`] call. This returns `Option<Post>` instead of throwing an error, which we can then use in our matching pattern. For additional methods of `QueryDsl` refer to the [`documentation`]
+
+
+[`.optional()`]: https://docs.diesel.rs/2.1.x/diesel/result/trait.OptionalExtension.html#tymethod.optional
+[`documentation`]: https://docs.diesel.rs/2.1.x/diesel/query_dsl/trait.QueryDsl.html
```

Comment:
I feel that's important to state here what's in that documentation:

```suggestion
Notice [`.optional()`] call. This returns `Option<Post>` instead of throwing an error, which we can then use in our matching pattern. For additional methods to modify the constructed select statements refer to the [`documentation`  of `QueryDsl`]


[`.optional()`]: https://docs.diesel.rs/2.1.x/diesel/result/trait.OptionalExtension.html#tymethod.optional
[`documentation` of `QueryDsl`]: https://docs.diesel.rs/2.1.x/diesel/query_dsl/trait.QueryDsl.html
```

Reply:
Thanks! Done, I added the commit 😄

<img width="1229" alt="Zrzut ekranu 2023-07-13 o 19 01 11" src="https://github.com/sgrif/diesel.rs-website/assets/37155981/d65a5a09-a6ae-4153-915a-a5684e1557fb">